src/Individual_XP.cpp

```diff
@@ -84,7 +84,7 @@ class Individual_XP : public PlayerScript
         if (PlayerXpRate* data = p->CustomData.Get<PlayerXpRate>("Individual_XP"))
         {
             uint32 rate = data->XPRate;
-            CharacterDatabase.DirectPExecute("REPLACE INTO `individualxp` (`CharacterGUID`, `XPRate`) VALUES (%u, %u);", p->GetGUID().GetCounter(), rate);
+            CharacterDatabase.DirectExecute("REPLACE INTO `individualxp` (`CharacterGUID`, `XPRate`) VALUES (%u, %u);", p->GetGUID().GetCounter(), rate);
```

Comment:
Here you forgot the same like on codeline 78.
If kept that way, the server will crash on user logout!

`CharacterDatabase.DirectExecute("REPLACE INTO `individualxp`(`CharacterGUID`,`XPRate`) VALUES ('%u', '%u');", p->GetGUID().GetCounter(), rate);`